libs/libarm-optimized-routines/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-option(USE_ARM_OPTIMIZED_ROUTINES "Use ARM optimized routines" ON)
+# TODO Enabling this feature causes "is missing exception specification" error on rockylinux8 and arm64
```

Comment:
Why not just remove those functions with issues?

Reply:
Is it useful if we can fix somehow?